build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id "java-library"
-    id "org.openmicroscopy.project" version "5.7.0"
-    id "org.openmicroscopy.gradle.ice-builder.slice" version "1.5.0"
+    id "org.openmicroscopy.project" version "5.7.2"
+    id "org.openmicroscopy.gradle.ice-builder.slice" version "1.5.4"
```

Comment:
Should this be rolled back to 1.5.0 given the issues that have been noted as part of https://github.com/ome/omero-blitz/pull/157